include/deal.II/dofs/dof_handler.h

```diff
@@ -701,6 +701,31 @@ class DoFHandler : public Subscriptor
   void
   distribute_mg_dofs();
 
+  /**
+   * Prepare all cells for p-adaptation. Only available in hp-mode.
+   *
+   * Does to future FE indices what
+   * Triangulation::prepare_coarsening_and_refinement() does to refinement
+   * flags.
+   *
+   * This function operates only on that part of the FE hierarchy
+   * registered with the hp::FECollection that contains the FE index
+   * @p contains_fe_index.
```

Comment:
I am not sure about the meaning of `contains_fe_index`. Is there other places in the documentation that explains this?